### ISSUE TITLE
Feature request: Support for Open Graph protocol stuff

### ISSUE THREAD

**Issue by adamzerner** [HermanMartinus/bearblog#75]
Check out https://opengraphprotocol.org/. The idea is that when you share posts in eg. Slack, there is a title, image preview, and description preview. It could be nice if blog authors were able to configure this. Eg. because currently, it takes the first `n` characters as it's description, which often don't give readers an accurate or compelling summary of what the article is about.

**Comment by HermanMartinus** [comment]
Yeah, I've been thinking about this as well. Will add it to my list of things-to-do

**Comment by HermanMartinus** [comment]
I've added the ability to set your own meta descriptions on both your homepage and your posts 921ad55e82f42013da39d8232a9773586242cc83. Adding meta images (and uploading images in-app instead of relying on Imgur) is coming soon.

**Comment by Cwpute** [comment]
Fantastic! tnaks :)

**Comment by sveggiani** [comment]
@HermanMartinus Revisiting this to avoid creating a new issue. Is there a way to override the OpenGraph image? 

I tried adding to the header directives, and the meta elements are in the page source code but they are superseded by the tags that Bearblog outputs.

<img width="1029" height="691" alt="Image" src="https://github.com/user-attachments/assets/0c86023c-dad3-4e45-978d-919ce3e8c254" />
